packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayout.tsx

```diff
@@ -145,32 +143,36 @@ const Left = () => {
                             transition: { duration: 0.3, bounce: 0 },
                         }}
                     >
-                        <Expander data-test="@welcome/title">
-                            <TrezorLogo type="symbol" width="57px" />
-                        </Expander>
-
-                        <LinksContainer>
-                            {isWeb() && (
-                                <TrezorLink type="hint" variant="nostyle" href={SUITE_URL}>
-                                    <Button
-                                        variant="tertiary"
-                                        icon="EXTERNAL_LINK"
-                                        iconAlignment="right"
-                                    >
-                                        <Translation id="TR_ONBOARDING_DOWNLOAD_DESKTOP_APP" />
-                                    </Button>
-                                </TrezorLink>
-                            )}
-                            <TrezorLink type="hint" variant="nostyle" href={TREZOR_URL}>
-                                <Button
-                                    variant="tertiary"
-                                    icon="EXTERNAL_LINK"
-                                    iconAlignment="right"
-                                >
-                                    trezor.io
-                                </Button>
-                            </TrezorLink>
-                        </LinksContainer>
+                        <TrafficLightOffset>
```

Comment:
Without this component it wasn't possible to move with window while user was on the Welcome screen 🫣